### PR TITLE
refactor: clarify save and load

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: encoders/nlp/TransformerTorchEncoder/tests/test_transformertorchencoder.py [hunk 1]
```diff
@@ -193,17 +189,19 @@ def test_wrong_layer_index(test_metas, layer_index):
     encoder.layer_index = layer_index
     test_data = np.array(['it is a good day!', 'the dog sits on the floor.'])
     with pytest.raises(ValueError):
-        encoded_data = encoder.encode(test_data)
+        _ = encoder.encode(test_data)
```
Comment (nan-wang) [inline encoders/nlp/TransformerTorchEncoder/tests/test_transformertorchencoder.py:192]: Any specific reason to remove the `save_config` and `assert` part?

Comment (JoanFM) [reply encoders/nlp/TransformerTorchEncoder/tests/test_transformertorchencoder.py:192]: because there is another test doing that

Comment (nan-wang) [reply encoders/nlp/TransformerTorchEncoder/tests/test_transformertorchencoder.py:192]: ok, i see.

**Review by nan-wang — APPROVED** [review]
LGTM👍

File: encoders/image/CustomImageTorchEncoder/tests/test_customimagetorchencoder.py [hunk 2]
```diff
@@ -33,46 +33,45 @@ def forward(self, x):
...
     output_dim = 10
     input_dim = 224
-    encoder = example_encoder(metas)
     test_data = np.random.rand(2, 3, input_dim, input_dim)
     encoded_data = encoder.encode(test_data)
     assert encoded_data.shape == (2, output_dim)
 
 
-def test_encoding_results_default(metas):
+def test_encoding_results_default(default_encoder):
```
Comment (nan-wang) [inline encoders/image/CustomImageTorchEncoder/tests/test_customimagetorchencoder.py:65]:
```suggestion
def test_encoding_results_default(encoder):
```

**Comment by nan-wang** [comment]
The test failed.
```
          HubIO@19[I]:    def test_encoding_results_default(default_encoder):
          HubIO@19[I]:        output_dim = 1280
          HubIO@19[I]:        input_dim = 224
          HubIO@19[I]:        test_data = np.random.rand(2, 3, input_dim, input_dim)
          HubIO@19[I]:>       encoded_data = encoder.encode(test_data)
          HubIO@19[I]:E       AttributeError: 'function' object has no attribute 'encode'
          HubIO@19[I]:
          HubIO@19[I]:tests/test_customimagetorchencoder.py:69: AttributeError

```

**Comment by nan-wang** [comment]
`encoder` -> `default_encoder`

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**